R/summary_rows.R

```diff
@@ -243,30 +243,28 @@ add_summary_location_row <- function(loc,
     if (df_type == "footnotes_df") {
 
       attr(data, df_type) <-
```

Comment:
Later, `add_location_row_*` methods might want to return data. (Move the `attr(data, df_type)` inside the function)

Reply:
This would be a nice change. Assigning to `attr(data, ...)` sometimes is inconsistent.

Reply:
Just made this an issue: https://github.com/rstudio/gt/issues/315